### PR TITLE
add several more Halloween themed clothes

### PR DESCRIPTION
**Pull request by LordBarkBread**
#### Summary
Content  "More Halloween items"

#### Purpose of change
Getting into the Halloween spirit and doing as I said I would on my last Halloween items PR I'm adding a few more pieces to celebrate the month of spooky!

#### Describe the solution
Added in a Steampunk Coat, Plague doctor Robes, Nun Habit and Nun Wimple, Pointed Hat variants, Witch Dress Variants, Plastic Pumpkin Mask, Leather Zombie Mask, and Plastic Mask Variants.

#### Describe alternatives you've considered
N/A

#### Testing
N/A

#### Additional context
This could still be a bit of a work in progress but as it stands now I'm happy with the few extra pieces in this PR and overall happy with the amount of Spooky Halloween items we have in game already. (Though I always like my costumes)

### CODE REVIEW

File: data/json/items/armor/coats.json [hunk 1]
```diff
@@ -3266,5 +3266,97 @@
     "warmth": 50,
     "material_thickness": 1,
     "flags": [ "VARSIZE", "POCKETS", "HOOD", "OUTER", "WATERPROOF", "SOFT" ]
+  },
+  {
+    "id": "trenchcoat_steampunk",
+    "type": "ARMOR",
+    "name": { "str": "steampunk trenchcoat" },
+    "description": "A thick black leather steampunk themed trenchcoat, unlike a regular trenchcoat this one has no pockets.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/coats.json:3274]:
```suggestion
    "description": "A thick black leather steampunk themed trenchcoat.  Unlike a regular trenchcoat, this one has no pockets.",
```

File: data/json/items/armor/coats.json [hunk 2]
```diff
@@ -3266,5 +3266,97 @@
...
+        "layers": [ "BELTED" ]
+      },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 100, "encumbrance": 12 }
+    ]
+  },
+  {
+    "id": "nun_habit",
+    "type": "ARMOR",
+    "name": { "str": "nun habit" },
+    "description": "A dark piece of clerical clothing.  Typically worn by christian nuns.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/coats.json:3338]:
```suggestion
    "description": "A dark piece of clerical clothing.  Typically worn by Christian nuns.",
```

File: data/json/items/armor/masks.json [hunk 3]
```diff
@@ -349,7 +350,45 @@
...
+        "id": "unicorn_plastic_mask",
+        "name": { "str": "plastic unicorn mask" },
+        "description": "It's a mask of a unicorn with a plastic piece sticking out as a horn.",
+        "weight": 15,
+        "append": true
+      },
+      {
+        "id": "bear_plastic_mask",
+        "name": { "str": "plastic bear mask" },
+        "description": "It's modeled after an american black bear.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/masks.json:380]:
```suggestion
        "description": "It's modeled after an American black bear.",
```

File: data/json/items/armor/scarfs.json [hunk 4]
```diff
@@ -780,6 +780,25 @@
     "flags": [ "OVERSIZE", "COLLAR", "ALLOWS_NATURAL_ATTACKS" ],
     "armor": [ { "encumbrance": 10, "coverage": 85, "covers": [ "head" ] } ]
   },
+  {
+    "id": "nun_wimple",
+    "type": "ARMOR",
+    "name": { "str": "wimple" },
+    "description": "A loose cloth headwrap and veil.  Typically worn by christian nuns.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/scarfs.json:787]:
```suggestion
    "description": "A loose cloth headwrap and veil.  Typically worn by Christian nuns.",
```

**Comment by Termineitor244** [comment]
Spooky!!

**Comment by Maleclypse** [comment]
Let me know when this is ready for review/merge?

**Pull request merged**